private/pkg/storage/bucket.go

```diff
@@ -52,6 +52,38 @@ type ReadBucket interface {
 	Walk(ctx context.Context, prefix string, f func(ObjectInfo) error) error
 }
 
+// PutOptions are the possible options that can be passed to a Put operation.
+type PutOptions struct {
+	IfNotExists      bool
+	CustomChunkSize  bool
+	ChunkSizeInBytes int
```

Comment:
`ChunkSize int64`, sizes are typically bytes

Reply:
Take care when casting. GCS uses `int` for chunk size.